scripts/test_schemas.sh

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env bash
 
-docker pull onsdigital/eq-schema-validator
-validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)"
+run_docker=true
+if [ "$1" == "-local" ] || [ "$2" == "-local" ]; then
```

Comment:
Would have been better to keep these changes in a separate PR, just for tracking purposes

Reply:
He just won't be told Shahi.

Reply:
Been bugging me for ages. I can separate it out I guess.